multiregion/scripts/jmx_metrics.sh

```diff
@@ -5,7 +5,7 @@ do
 
   echo -e "\n\n==> Monitor $metric \n"
 
-  for topic in single-region multi-region-sync multi-region-async multi-region-default
+  for topic in single-region multi-region-sync multi-region-async multi-region-default under-min-isr-promotion under-replicated-promotion leader-is-observer-promotion
```

Comment:
@thomaskwscott missing modification above to add `ObserversInIsrCount` in line 3, to the list of metrics.

1. Please add metric to this script
2. Corresponding change needs to be made in the docs which shows the expected output

Reply:
updated.